code/modules/mining/mine_items.dm

```diff
@@ -529,7 +529,9 @@
 
 /obj/structure/fans/Destroy()
 	arbitraryatmosblockingvar = 0
-	air_update_turf(1)
+	var/turf/T = get_turf(src)
+	spawn(0)
+		T.air_update_turf(TRUE)
```

Comment:
This isn't even necessary. Fans work fine and activate atmos just fine.

They set `arbitraryatmosblockingvar`, which is what `CanAtmosPass` determines if air can pass or not.

Reply:
Overlooked that var being set. Reverted it now due to it not being needed in that case.